_includes/tutorials/anomaly-detection/ksql/markup/dev/set-properties.adoc

```diff
@@ -0,0 +1,5 @@
+Set ksqlDB to process data from the beginning of each Kafka topic:
```

Comment:
Given that there is no data in the topic yet, is this required?

Reply:
Yes it is required. I ran through the tutorial without it, the select statements consume from latest. With no data coming in, the derived streams wont' have any data in them.